### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by leonardodiasb**
- Used CSS to style the page based on the Bookstore CMS project page on Zeplin.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ const Navbar = () => {
 
   return (
     <nav>
-      <h1>Bookstore CMS</h1>
+      <h1 className="logo">Bookstore CMS</h1>
```
Comment (ZawadiSumba66) [inline src/components/Navbar.js:21]: - [x] The bookstore and the cms words ought to appear in one line. Kindly fix this.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -17,7 +18,7 @@ const Navbar = () => {
...
       <ul className="nav-list">
         {links.map((link) => (
           <li key={link.id} className="nav-li">
```
Comment (ZawadiSumba66) [inline src/components/Navbar.js]: - [x] The books link in the navbar has a darker shade than the categories link to the right of it as you can see below:
image

File: src/components/books/Book.js [hunk 3]
```diff
@@ -12,23 +12,27 @@ const Book = ({ title, author }) => {
...
           <button type="button">Comments</button>
+          <vr className="vr" />
           <button type="button" onClick={removeBookFromStore}>Remove</button>
+          <vr className="vr" />
           <button type="button">Edit</button>
         </div>
       </div>
       <div className="progress-info">
-        <p className="percentage">0% Completed</p>
+        <span className="Percent-Complete">0%</span>
```
Comment (ZawadiSumba66) [inline src/components/books/Book.js]: - [x] The progress chart to the left of the percentage number is currently missing while displaying a book like the one as follows:
image

File: src/components/books/InputBook.js [hunk 4]
```diff
@@ -17,11 +17,11 @@ const InputBook = () => {
...
-    <form onSubmit={submitBookToStore}>
-      <h2>ADD NEW BOOK</h2>
-      <input name="title" placeholder="Book Title" />
-      <input name="author" placeholder="Author" />
-      <button type="submit">ADD BOOK</button>
+    <form className="input-form" onSubmit={submitBookToStore}>
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <input className="input-title" name="title" placeholder="Book Title" />
+      <input className="input-author" name="author" placeholder="Author" />
+      <button className="add-btn" type="submit">ADD BOOK</button>
```
Comment (ZawadiSumba66) [inline src/components/books/InputBook.js]: - [x] Kindly ensure that the book, author inputs, and the add button are all inline. The add and author input should not overflow to the next line. 
- [x] Also ensure that the background color for the inputs is white. 
- [x] I would also urge to try and add some margin below the form.
- [ ] Check the screenshots below for comparison in accordance to the above suggestions:
yours:
image
=======================================================================
template's
image

File: src/index.css [hunk 5]
```diff
@@ -25,3 +42,303 @@ code {
     'Courier New',
     monospace;
 }
+
+html {
+  background-color: #fafafa;
+}
+
+nav {
```
Comment (ZawadiSumba66) [inline src/index.css:20]: - [ ] There is a bit of redundant code in your CSS file. You could reduce this by adding a common class where a particular class applies to different JSX tags.
For example  
+ font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  font-family: Montserrat, sans-serif; and a lot more selectors.
You can check [this article for support](https://speckyboy.com/good-bad-css-practices/)

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi @leonardodiasb 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @leonardodiasb ! 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## **[- NOTE FOR THE NEXT REVIEWER -]**

**As long as I can see, he did all the fixes, and the only thing left is the linters, please take it in consideration and approve this project if all the linters pass**

attention

 ## Required Changes ♻️

- [ ] Please, double-check the linters, you have some errors from your last change, remember that all the linters must pass in order to approve this project.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @leonardodiasb  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**